content/api/apps/data-elements/user-defined-metadata/_index.en.md

```diff
@@ -0,0 +1,65 @@
+---
+title: User defined metadata
+description: App API for adding user defined metadata to data elements.
+toc: true
+tags: [api]
+weight: 100
+---
+
+## Overview
+
+Custom metadata on data elements allows key-value pairs of metadata to be added to a data element via the API. There are dedicated endpoints for setting and retrieving these values. 
+The data is also available in other endpoints where metadata about data elements is retrieved.
+
+It is important to note that this is data that users with write permissions on the data element can freely edit via the API. 
```

Comment:
Could use `{{< panel info >}}` to convey the importance instead of "it is important to note that .."